spec/integration/post_index_spec.rb

```diff
@@ -16,33 +16,44 @@
                          comments_counter: 0, author_id: @user1.id)
     @post5 = Post.create(title: 'Automobile engineering', text: 'Reading', likes_counter: 0,
                          comments_counter: 0, author_id: @user2.id)
-    Comment.create(text: 'This is great', post_id: @post1.id, author_id: @user2.id)
+    @comment1 = Comment.create(text: 'This is great', post_id: @post1.id, author_id: @user2.id)
+    visit user_path(@user1.id)
   end
 
   context 'Load index page' do
-    it 'Should see the posts title' do
-      visit posts_path(@user1.id)
-      expect(page).to have_content('This is a post')
+    it 'Should see the profile picture' do
+      expect(page).to have_selector('img')
     end
 
     it 'Should see the posts author' do
-      visit posts_path(@user1.id)
       expect(page).to have_content('Aisha')
     end
 
```

Comment:
**Tests that assert you can see a section for pagination if there are more posts than fit on the view.**

- [ ] It would be very nice if you make test for the above scenario as suggested by the previous reviewer to make your application tested and save it from unexpected breaks :+1: